ci/prow/config.yaml

```diff
@@ -5772,11 +5772,11 @@ periodics:
         -label:lifecycle/rotten"
       - "--updated=2160h"
       - "--token=/etc/housekeeping-github-token/token"
-      - "--comment=Issues go stale after 90 days of inactivity.<br/>
-        Mark the issue as fresh by adding the comment <code>/remove-lifecycle stale</code>.<br/>
-        Stale issues rot after an additional 30 days of inactivity and eventually close.<br/>
-        If this issue is safe to close now please do so by adding the comment <code>/close</code>.<br/><br/>
-        Send feedback to [Knative Productivity Slack channel](https://knative.slack.com/messages/CCSNR4FCH) or file an issue in [knative/test-infra](https://github.com/knative/test-infra/issues/new).<br/><br/>
+      - "--comment=Issues go stale after 90 days of inactivity.\n
+        Mark the issue as fresh by adding the comment \"/remove-lifecycle stale\".\n
```

Comment:
Yes. That's why I mentioned using a regular string with LFs instead of a raw string.

Reply:
Sorry I misunderstood what you meant.
However, if I use double-quoted strings, I'll have to write like below:
```
comment := "a....\n" +
    fmt.Sprintf("b...\n", xxx) +
    "c...\n" +
    fmt.Sprintf("d...\n", yyy)
```
Otherwise I will get a really long string. See https://programming.guide/go/multiline-string.html.

I feel this format harder to read than the raw string. But if you are okay with it I can make this change.

Reply:
Make it a long multi-line format string; concatenating strings and Sprintf like above is  indeed hard to read.

Reply:
You meant something like below, right?
```
staleComment := fmt.Sprintf("--comment=Issues go stale after %d days of inactivity.\\n\n"+
	"        Mark the issue as fresh by adding the comment `/remove-lifecycle stale`.\\n\n"+
	"        Stale issues rot after an additional %d days of inactivity and eventually close.\\n\n"+
	"        If this issue is safe to close now please do so by adding the comment `/close`.\\n\\n\n"+
	"        %s\\n\\n\n"+
	"        /lifecycle stale", daysToStale, daysToRot, feedbackNote)
```

Just want to make sure before I make the changes.

Reply:
Yes

Reply:
Just note that you don't double escape \n in strings, and please make sure it does work due to the blank line supressing in the config generation.

Reply:
Done.
Examples in https://github.com/Fredy-Z/Algorithms/issues/3#issuecomment-534669013

Reply:
Beautiful!